contrib/externs/chrome_extensions.js

```diff
@@ -11016,3 +11016,72 @@ chrome.downloads.DeterminingFilenameEvent.prototype.hasListeners =
  * https://developer.chrome.com/extensions/downloads#event-onDeterminingFilename
  */
 chrome.downloads.onDeterminingFilename;
+
+
+/**
+ * @const
+ * @see https://developer.chrome.com/extensions/declarativeContent
+ */
+chrome.declarativeContent = {};
+
+
+/** @type {string} */
+chrome.declarativeContent.PageStateMatcherInstanceType;
+
+
+/** @type {string} */
+chrome.declarativeContent.ShowPageActionInstanceType;
+
+
+/** @type {string} */
+chrome.declarativeContent.SetIconInstanceType;
+
+
+/** @type {string} */
+chrome.declarativeContent.RequestContentScriptInstanceType;
+
+
+/**
+ * @typedef {!{
+ *   pageUrl: (!UrlFilter|undefined),
+ *   css: (!Array<string>|undefined),
+ *   isBookmarked: (boolean|undefined)
+ * }}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-PageStateMatcher
+ */
+chrome.declarativeContent.PageStateMatcher;
+
+
+/**
+ * @constructor
```

Comment:
Do you ever actually write code that says `new ShowPageAction()`, or do you just need to declare parameters and variables with this type? If you never say `new`, then please replace `@constructor` with `@record` here to make this an interface type.

Please ask yourself the same question for all the other new types you've added and update as appropriate.
Thanks.